PlayCover/Views/Uninstaller.swift

```diff
@@ -101,18 +101,19 @@ class Uninstaller {
                     UninstallPreferences.shared.showUninstallPopup = false
                 }
 
-                uninstall(app)
+                Task { await uninstall(app) }
             }
         } else {
-            uninstall(app)
+            Task { await uninstall(app) }
         }
     }
 
-    static func uninstall(_ app: PlayApp) {
+    static func uninstall(_ app: PlayApp) async {
         var uninstallNum = 0
 
+        UninstallVM.shared.next(.begin, 0.0, 0.0)
```

Comment:
There should be stages to the uninstall progress bar, similar to the other progress vms for a better user experience. You could also consider adding the uninstall vm to `clearAllCache` instead, as that is the one that consumes the most time, and progress could be easily tracked as it goes through a set amount of directories.

Reply:
Based on my observation, it seems that 'clearAppData' is the only task that consumes time. Other tasks are completed instantly, and even if suitable stage values are set for each tasks, it will not make a difference to the visual presentation of the UI at all.💦

However, I am currently attempting to modify the interface style for clearAllCache to match the native alert with progress bar style used by uninstaller. This is a good idea.